app/models/user.rb

```diff
@@ -49,71 +49,92 @@ def should_generate_new_friendly_id?
   source: :movie
 
   def all_lists
-    (self.lists | self.member_lists).uniq
+    list_ids = lists.pluck(:id)
+    member_list_ids = member_lists.pluck(:id)
+    ids = (list_ids + member_list_ids).uniq
```

Comment:
This query, `all_listings` and `all_movies` use multiple queries.

Given the way we have tables set up, I don't know of any other way to accomplish it without making a couple queries.

Even if there is some way to do it, it's probably more efficient to do a couple simple queries than a huge query that joins several tables.